xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Visualization.h

```diff
@@ -94,26 +94,63 @@ typedef struct AddonInstance_Visualization /* internal */
   KodiToAddonFuncTable_Visualization toAddon;
 } AddonInstance_Visualization;
 
+//============================================================================
+/// \defgroup cpp_kodi_addon_visualization_VisTrack class VisTrack
+/// \ingroup cpp_kodi_addon_visualization
+/// @brief **Visualization track information structure**
+///
+/// Becomes given on kodi::addon::CInstanceVisualization::UpdateTrack() call to have on
+/// information of currently played song.
+///
+//@{
 struct VisTrack
 {
+  /// @brief Title of the currently playing song
   const char *title;
+
+  /// @brief Get artist names from the artist decription string (if it exists)
+  /// as a single string
   const char *artist;
+
+  /// @brief Album from which the current song is from
   const char *album;
+
+  /// @brief Album artist names from the artist decription string (if it
+  /// exists) as a single string.
   const char *albumArtist;
+
+  /// @brief Shows the genre of the currently playing artist
```

Comment:
Thanks a lot for your review and the request!

Have compared wird Python interface there is named "... the genre name from music tag as string if present." and also looked to the source of them.

With Artist is then wrong from me.